Sources/Keyboard/Layouts/Piano.swift

```diff
@@ -51,14 +50,14 @@ struct Piano<Content>: View where Content: View {
         }
     }
 
-    func whiteKeyWidth(size: CGSize) -> CGFloat {
-        size.width / CGFloat(whiteKeys.count)
+    func whiteKeyDimension(_ dimension: CGFloat) -> CGFloat {
```

Comment:
I assume this name change is because width implies horizontal, but dimension is just too vague.

Reply:
Perhaps we stick with `width`, that means how wide the keyboard key is from side to side.

<img width="392" alt="Screenshot 2022-10-23 at 9 05 45 AM" src="https://user-images.githubusercontent.com/4270232/197368293-c5af1cef-d7d8-40af-a577-4e9979cedba7.png">

Reply:
Agreed